docs/ops.html

```diff
@@ -3895,7 +3895,13 @@ <h3>Migrating brokers to KRaft</h3>
   <h3>Reverting to ZooKeeper mode During the Migration</h3>
     While the cluster is still in migration mode, it is possible to revert to ZK mode. In order to do this:
     <ol>
-      <li>One by one, take each KRaft broker down. Remove the __cluster_metadata directory on the broker. Then, restart the broker in ZooKeeper mode.</li>
+      <li>
+        One by one, take each KRaft broker down.
+        Remove the __cluster_metadata directory on the broker.
+        Remove the zookeeper.metadata.migration.enable flag and the KRaft controllers related configuration like controller.quorum.voters
+        and controller.listener.names from the broker configuration properties file.
+        Then, restart the broker in ZooKeeper mode.
+      </li>
```

Comment:
Could we use `<li>` tag for each line?

Reply:
- The way it's written sounds like you first need to shut down all brokers. Instead you need to run the steps on one broker at a time. 
- Can you also use `<code>` tags around configuration names?
- `flag` -> `configuration`
- `configuration like controller.quorum.voters` -> `configurations like controller.quorum.voters`

Reply:
So we are already in a ordered list <ol>, where the first step (about shutting down brokers) should be broken in other sub-steps. So I made the change by having the list iteam <li> including an unordered list with the sub-steps. Let me know what you think this way.